src/hatti/ona/forms.cljs

```diff
@@ -215,9 +215,20 @@
                                           first)
                               formatted (get-label option language)]
                           (or formatted answer))
-    (select-all? field) (let [names (set (string/split answer #" "))]
-                          (->> (:children field)
-                               (filter #(contains? names (field-key %)))
+    (select-all? field) (let [names (set (string/split answer #" "))
+                              appearance-value (-> field :control :appearance)
+                              uses-search-expression?
+                              (and (string? appearance-value)
```

Comment:
let's move this to a helper function, since that's the purpose it's service here